WMIAdventure/frontend/src/js/components/battle/organisms/PostBattle/PostBattle.js

```diff
@@ -37,8 +37,16 @@ class PostBattle extends React.Component {
     opponentWinHandler = () => this.props.win === null ? null : !this.props.win
 
     updateUserLevel = async () => {
-        if (!this.opponentWinHandler())
-            await this.props.attacker.fetchNonVitalDataFromBackend(true);
+        if (!this.opponentWinHandler()) {
+            if (this.props.attacker) {
+                const levelBefore = this.props.attacker.level;
+                await this.props.attacker.fetchNonVitalDataFromBackend(true);
+                if (levelBefore < this.props.attacker.level) {

```

Comment:
Czy ten if w ogóle jest potrzebny? Jest tutaj tylko console log i alert, a alerta bym usunął.
Sama animacja level upa, ten powiększający się numerek i złoty błysk jest spoko.

Reply:
a kurde zapomniałem to usunąć